tests/save-and-return.js

```diff
@@ -0,0 +1,16 @@
+import test from 'ava'
+import withPage from '../utils/withPage'
+import config from '../config'
+
+test(
+  'Save & return button is displayed to the user',
```

Comment:
How are we testing that save and return button is not being displayed to user? Perrsonally I think we need more than one form...

Reply:
Good point. Either another form or a way to change settings on the fly. We now have two settings... 1) save & return, 2) user attachments, that need to be turned on and off for testing each one.